zio-http-example/src/main/scala/example/PlainTextBenchmarkServer.scala

```diff
@@ -30,11 +30,11 @@ object PlainTextBenchmarkServer extends ZIOAppDefault {
     .withServer(STATIC_SERVER_NAME)
     .freeze
 
-  private def plainTextApp(response: Response) =
-    Http.fromHExit(HExit.succeed(response)).whenPathEq(plaintextPath)
+  private def plainTextApp(response: Response): HttpRoute[Any, Nothing] =
+    Handler.response(response).toRoute.whenPathEq(plaintextPath)
```

Comment:
`toRoute`, some old temrinology still exists??

Reply:
Fixed

Reply:
although I think `.toRoute` was a nicer name than `.toHttp` (and `Route` vs `Handler` is a better distinction than `Http` vs `Handler` which does not make much sense to me)